be/src/olap/olap_common.h

```diff
@@ -143,7 +142,8 @@ enum FieldType {
     OLAP_FIELD_TYPE_HLL = 23,
     OLAP_FIELD_TYPE_BOOL = 24,
     OLAP_FIELD_TYPE_OBJECT = 25,
-    OLAP_FIELD_TYPE_STRING = 26
+    OLAP_FIELD_TYPE_STRING = 26,
+    OLAP_FIELD_TYPE_SIZE = 27 // Size of FieldType, Notice: This must be the last field of FieldType
```

Comment:
May be we should enlarge this value to make sure it can be alway at the last field in enum.

Reply:
ok,i try to delete this ugly param